### PR TITLE
Fix win tether

### PR DESCRIPTION
**Pull request by hickeng**
This fixes the windows tether compilation issues, and tests that failed
now we actually use the utils package. Also fixes up the bootstrap iso
setup to match the tether behaviour.

### CODE REVIEW

File: cmd/tether/utils/hostname_linux.go [hunk 1]
```diff
@@ -23,6 +25,8 @@ import (
 	"github.com/vmware/vic/pkg/trace"
 )
 
+var hostnameFile = "/etc/hostname"
```
Comment (caglar10ur) [inline cmd/tether/utils/hostname_linux.go]: const?

File: cmd/tether/utils/interfaces_linux.go [hunk 2]
```diff
@@ -24,8 +26,12 @@ import (
 
 	"github.com/vishvananda/netlink"
 	"github.com/vmware/vic/metadata"
+	"github.com/vmware/vic/pkg/trace"
 )
 
+var hostsFile = "/etc/hosts"
+var resolvFile = "/etc/resolv.conf"
```
Comment (caglar10ur) [inline cmd/tether/utils/interfaces_linux.go]: const?

**Comment by caglar10ur** [comment]
Overall LGTM

**Comment by fdawg4l** [comment]
LGTM

File: cmd/tether/utils/hostname_linux.go (hunk 1 above)
Comment (hickeng) [inline cmd/tether/utils/hostname_linux.go]: wanted to leave it open for being modified by tests if I could find a neat way of doing so

**Comment by caglar10ur** [comment]
LGTM :+1: :sheep: :it:

**Pull request merged**